backend/inspirehep/assign/tasks.py

```diff
@@ -0,0 +1,141 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+import structlog
+from celery import shared_task
+from inspire_schemas.builders import LiteratureBuilder
+from invenio_db import db
+from invenio_pidstore.errors import PIDDoesNotExistError
+from jsonschema import ValidationError
+from sqlalchemy.exc import (
+    DisconnectionError,
+    OperationalError,
+    ResourceClosedError,
+    TimeoutError,
+    UnboundExecutionError,
+)
+from sqlalchemy.orm.exc import NoResultFound, StaleDataError
+
+from inspirehep.records.api import ConferencesRecord, LiteratureRecord
+from inspirehep.records.errors import MissingArgumentError
+
+LOGGER = structlog.getLogger()
+
+
+@shared_task(
+    ignore_results=False,
+    queue="assign",
+    acks_late=True,
+    retry_backoff=2,
+    retry_kwargs={"max_retries": 6},
+    autoretry_for=(
+        NoResultFound,
+        StaleDataError,
+        DisconnectionError,
+        TimeoutError,
+        UnboundExecutionError,
+        ResourceClosedError,
+        OperationalError,
+    ),
```

Comment:
aren't we use this errors everywhere? can we move them and reuse the same?

Reply:
We use similar ones in migrator tasks, they are not 100% identical, so I would have to create a common group of exceptions and manually add rest of them in proper places. It's not used that often to be worth additional complexity IMHO.